lib/CommandResult.js

```diff
@@ -65,8 +65,17 @@ class CommandResult {
   _getHumanReadableResult() {
     if (this._humanReadableFormatIsTable()) {
       const tableData = this._getTableData();
-      const formattedTableData = consoleTable.getTable(tableData);
-      return Array.isArray(tableData) ? `Count: ${tableData.length}${EOL}${EOL}${formattedTableData}` : formattedTableData;
+      const isList = Array.isArray(tableData);
+      const sortData = isList && !isEmpty(tableData) && Object.prototype.hasOwnProperty.call(tableData[0], 'name');
```

Comment:
I think this code should not be here. If you want something ordered, you should do it in the command handler itself, not here.

Reply:
We need this for all lists, don't we? This is why I've put it here, it seems common logic to me.

Reply:
It is common, but this doesn't mean it belongs here. The normal place to do sorting is where you know what your data is, hence know what to sort it by. What if I sorted my data in the handler and it happens to have a "name" property in the items? How do I sort it by something different?
I think this belongs in a helper method that gets called whenever you need it, but not in the common place where all data goes through.

Reply:
The moment you need to sort by a different property or not sort at all, you can add a method to the CommandResult that sets that property IMO.
Anyways, I'll add a helper method in Utils.